bin/platformsh/pre_start-per_node-clear_cache.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# This script gets run as part of the .platform.app.yaml deploy step
+# On PE Cluster (usually just production) this should be setup by P.sh team as part of pre_start event
+
+set -e
+
+#date
+echo "removing var/cache/${SYMFONY_ENV}/*.* to avoid Symfony container issues on interface changes"
+rm -Rf var/cache/${SYMFONY_ENV}/*.*
+#date
+echo "clearing application cache"
+bin/console cache:clear
```

Comment:
you use `php bin/console` in some places but not here, is it normal?

Reply:
It doesn't matter as `bin/console` has `#!/usr/bin/env php`, but let's try to be consistent nevertheless 👍 :

```suggestion
php bin/console cache:clear
```